docs/resources/azure_network_watcher.md.erb

```diff
@@ -0,0 +1,84 @@
+---
+title: About the azure_network_watcher Resource
+platform: azure
+---
+
+# azure\_network\_watcher
+
+Use the `azure_network_watcher` InSpec audit resource to test properties
+of an Azure Network Watcher.
+
+<br />
+
+## Syntax
+
+An `azure_network_watcher` resource block identifies a Network Watcher by
+name and resource group.
+
+    describe azure_network_watcher(resource_group: 'example', name: 'WatcherName') do
+      ...
+    end
+<br />
+
+## Examples
+
+### Test that an example Resource Group has the specified Network Watcher
+
+    describe azure_network_watcher(resource_group: 'example', name: 'WatcherName') do
+      it { should exist }
+    end
+
+### Test that an example Resource Group has a Network Watcher with the correct Provisioning State
```

Comment:
The examples should be a little less dry. In this one, we're specifically testing that the watcher was successfully provisioned, so let's say that instead of the more generic "correct provisioning state".